src/Microsoft.AspNet.Mvc.Razor/RazorPage.cs

```diff
@@ -846,6 +893,33 @@ public virtual HtmlString SetAntiforgeryCookieAndHeader()
             return HtmlString.Empty;
         }
 
+        private bool IsSingleBoolOrNullValue(AttributeValue[] values)
```

Comment:
Maybe rename it to something like `IsSingleBoolFalseOrNullValue` (not a good name. I know). But the current name says it will return true for any bool.